cmake/helpers.cmake

```diff
@@ -232,9 +232,8 @@ function(add_umf_target_compile_options name)
             PRIVATE -fPIC
                     -Wall
                     -Wextra
-                    -Wpedantic
                     -Wformat-security
-                    -Wcast-qual
+                    -Wno-cast-qual
```

Comment:
Why do you disable this flag?

Reply:
In few cases there are const qualifier drop. If you have an idea to mitigate it, I'm open to discussion.

Reply:
@KFilipek please add TODO

Reply:
After a merge, I can add issue for that, if you want.